ui/user/src/lib/services/chat/types.ts

```diff
@@ -692,7 +690,5 @@ export interface MCPCatalogServer extends MCPCatalogEntryServerManifest {
 	createdAt: string;
 	updatedAt: string;
 	type: string;
-	metadata?: {
-		categories?: string;
-	};
+	manifest: MCPCatalogEntryServerManifest;
```

Comment:
I'm not sure that this is correct. Iirc, this type has the `fixedURL` and `hostname` fields, whereas an actual MCPServer (not a catalog entry), which is what I believe this type represents, only has `url`.

Here is the type definition for the MCPServerCatalogEntryManifest: https://github.com/obot-platform/obot/blob/9b5253fe5ab7852e87318e489091892634b89333/apiclient/types/mcpserver.go#L13

And for the MCPServerManifest: https://github.com/obot-platform/obot/blob/9b5253fe5ab7852e87318e489091892634b89333/apiclient/types/mcpserver.go#L47